server/sonar-ce/src/test/java/org/sonar/ce/taskprocessor/CeWorkerFactoryImplTest.java

```diff
@@ -22,7 +22,9 @@
 
 import java.util.HashSet;
 import java.util.Set;
+import org.junit.Rule;
```

Comment:
hum... this change is weird. New imports are added but there are no change to the code.